tsl/test/expected/gapfill.out

```diff
@@ -1634,6 +1634,21 @@ FROM metrics_tstz WHERE time < '2017-01-01' GROUP BY 1;
  Sun Jan 01 16:00:00 2017 PST |  1.66666666666667
 (3 rows)
 
+SELECT
+  time_bucket_gapfill('12h'::interval,time,'2017-01-01'::timestamptz, '2017-01-02'::timestamptz),
+  interpolate(
+    avg(v1),
+    (SELECT ('2017-01-01'::timestamptz,1::float)),
+    (SELECT ('2017-01-02'::timestamptz,2::float))
+  )
+FROM metrics_tstz WHERE time_bucket_gapfill('12h'::interval,time,'2017-01-01'::timestamptz, '2017-01-02'::timestamptz) < '2017-01-01' GROUP BY 1;
```

Comment:
Can `ORDER BY` be added to the query to avoid flakiness? Or will it affect the optimisation?

Reply:
I think it should be fine for this one